apps/admin-gui/src/app/shared/components/services-list/services-list.component.html

```diff
@@ -34,8 +46,14 @@
           <td *matCellDef="let service" mat-cell>{{service.name}}</td>
         </ng-container>
         <ng-container matColumnDef="enabled">
-          <th *matHeaderCellDef mat-header-cell>{{'SHARED.COMPONENTS.SERVICES_LIST.ENABLED' | translate}}</th>
-          <td *matCellDef="let service" mat-cell>{{service.enabled}}</td>
+          <th *matHeaderCellDef mat-header-cell></th>
+          <td *matCellDef="let service" mat-cell><mat-icon
+            class="align-bottom"
+            *ngIf="service.enabled"
+            matTooltip="{{'SHARED.COMPONENTS.SERVICES_LIST.ENABLED_HINT' | translate}}">
+            done
+          </mat-icon>
```

Comment:
would be nice if there was some icon for disabled. Try to find it here https://material.io/resources/icons/?icon=home&style=baseline. From there we have most of our icons. Also if you find some icon more appropriate for enabled it would be nice.